src/userApp/controllers.js

```diff
@@ -0,0 +1,64 @@
+/* c8 ignore next 80 */
```

Comment:
There is no need to ignore the file that is already tested extensively.

Reply:
my fault. thanks